pkgs/build-support/php/builders/v2/hooks/composer-vendor-hook.sh

```diff
@@ -96,9 +96,23 @@ composerVendorInstallHook() {
     mkdir -p $out
 
     cp -ar composer.json $(composer config vendor-dir) $out/
+    mapfile -t installer_paths < <(jq -r '(.extra."installer-paths" // {}) | keys[]' composer.json)
```

Comment:
Where is `mapfile` coming from? What if `.extra` or `.extra.installer-paths` does not exists?

Reply:
1. mapfile is a bash builtin command
2. It's already handled by the `// {}` part

Reply:
1. fait enough, TIL
2. You could use `try` so that it is consistent with the rest ? (https://github.com/coffeeispower/nixpkgs/blob/08522a1ed6f73bc24ba733d5e3ee27b6ca2e86a3/pkgs/build-support/php/builders/v2/hooks/composer-install-hook.sh#L63)